### PR TITLE
Update DomPdfEngine.php

### PR DESCRIPTION
**Pull request by Iteracode**
Dompdf now works correctly with Composer.

### CODE REVIEW

File: src/Pdf/Engine/DomPdfEngine.php [hunk 1]
```diff
@@ -29,7 +30,7 @@ public function __construct(CakePdf $Pdf)
      */
     public function output()
     {
-        $DomPDF = new \DOMPDF();
+        $DomPDF = new DOMPDF();
```
Comment (ADmad) [inline src/Pdf/Engine/DomPdfEngine.php]: Can you please also change this to correct casing `Dompdf`.

**Comment by Iteracode** [comment]
@ADmad  Like this ?

**Comment by ADmad** [comment]
Yup. Can you please squash the commits into one?

**Comment by ADmad** [comment]
Fixed on master

**Pull request closed without merging**